packages/0x.js/src/utils/class_utils.ts

```diff
@@ -0,0 +1,17 @@
+import * as _ from 'lodash';
+
+export const classUtils = {
+    bindAll(self: any, exclude: string[] = ['contructor'], thisArg?: any): void {
```

Comment:
Let's add a one-liner comment above this method explaining what it does and when using it might be useful.